First open api

Lipek88 opened the pull request.

Lipek88 committed:
first try with openApi

Lipek88 committed:
open api partial

Lipek88 committed:
open api compilation failure

Lipek88 committed:
open api compilation success

DrJul commented on pom.xml:
```diff
@@ -45,6 +45,25 @@
+			<version>2.5</version>
+			<scope>provided</scope>
+		</dependency>
+
```
dodaj zależność:

<dependency>
			<groupId>io.swagger.core.v3</groupId>
			<artifactId>swagger-annotations</artifactId>
			<version>2.2.10</version>
		</dependency>

DrJul commented on pom.xml:
```diff
@@ -61,6 +80,55 @@
 					</excludes>
 				</configuration>
 			</plugin>
+<!--			<plugin>-->
```
usuń zakomentowany kod

DrJul commented on src/main/resources/openApi.yaml:
```diff
@@ -0,0 +1,93 @@
+
+components:
+  schemas:
+    ProblemTo:
```
Mamy już obiekty o takich nazwach w apce, lepiej unikać takich sytuacji i dodać jakiś sufiks

Lipek88 committed:
add dependencies,
delete commented code
rename ProblemTo using by OpenApi
rename package name into to
